st2common/st2common/services/triggers.py

```diff
@@ -245,7 +245,7 @@ def create_trigger_db(trigger_api):
     return trigger_db
 
 
-def create_or_update_trigger_db(trigger):
+def create_or_update_trigger_db(trigger, log_not_unique_error_as_debug=False):
     """
```

Comment:
Note: We want to default this to ``False`` everywhere because in other context, this error would indeed be fatal.